applications.ttl

```diff
@@ -26,7 +26,7 @@
 source: a schema:CreativeWork ;
   schema:comment """Description of OpenLink Software Applications and VAD Packages."""^^xsd:string ;
   schema:dateCreated  "2014-07-03T15:44:00-05:00"^^xsd:dateTime ;
-  schema:dateModified "2021-04-09T10:00:00-05:00"^^xsd:dateTime ;

```

Comment:
@openlink @HughWilliams 

This is the only line that appears to have been changed by this PR -- describing the `applications.ttl` document, but not describing FCT, R2RML, nor Conductor VADs.  Is that as intended?